### PR TITLE
Proofing + fixed small typo

### PR DESCRIPTION
**Pull request by guywald1**
Small PR here.

P.S. I absolutely love Webpack, so I'd like to take this opportunity to thank all you superheroes from the bottom of my heart.

### CODE REVIEW

File: content/guides/shimming.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ contributors:
...
+`webpack` as a module bundler can understand modules written as ES2015 modules, CommonJS or AMD. But many times, while using third party libraries, we see that they expect dependencies which are global, AKA `$` for `jquery`. They might also be creating global variables which need to be exported. Here we will see different ways to help webpack understand these __broken modules__.
 
 ## Prefer unminified CommonJS/AMD files over bundled `dist` versions.
 
-Most modules link the `dist` version in the `main` field of their `package.json`. While this is useful for most developers, for webpack it is better to alias the src version because this way webpack is able to optimize dependencies better. However in most cases `dist` works fine as well.
+Most modules link the `dist` version in the `main` field of their `package.json`. While this is useful for most developers, for Webpack it is better to alias the src version because this way webpack is able to optimize dependencies better. However in most cases `dist` works fine as well.
```
Comment (bebraw) [inline content/guides/shimming.md]: It's "webpack" in these docs always. We need a linting rule for that. :)

Comment (guywald1) [inline content/guides/shimming.md]: Noted, thank you :)

Comment (bebraw) [inline content/guides/shimming.md]: Can you change this? Good to merge after that.

Comment (guywald1) [inline content/guides/shimming.md]: Of course. Changed.

**Pull request merged**